Tasks/Common/RemoteDeployer/RunPowerShellScriptJob.ps1

```diff
@@ -0,0 +1,122 @@
+$ExecutePsScript = {
+    Param(
+        [string] $scriptPath,
+        [string] $scriptArguments = "",
+        [string] $inlineScript,
+        [bool] $inline,
+        [string] $workingDirectory = "",
+        [string] $_errorActionPreference = "Continue",
+        [bool] $ignoreLASTEXITCODE,
+        [bool] $failOnStdErr,
+        [string] $initializationScriptPath,
+        [string] $sessionVariables
+    )
+
+    $Global:ErrorActionPreference = "Continue";
```

Comment:
this is tricky, if something went wrong difficult to find the exact error, consider it as chain of errors